luarules/gadgets/unit_type_checker.lua

```diff
@@ -0,0 +1,62 @@
+
+UTC = {}
```

Comment:
If a global variable is absolutely needed, use the `GG` gadget global table. Avoid use of acronyms, only people who already know what they mean know what they mean.

e.g. `GG['UnitTypeChecker']`

Reply:
Understandable, I think this part might be redundant now in any case